### DISCUSSION TITLE
ssh config proxycommand query

### DISCUSSION THREAD

**Discussion by uvarvu** [iterate-ch/cyberduck#14389] [Q&A]
Cyberduck appears to ignore this SSH config which I use to connect to AWS ec2-instances on the command line via SSM.  Do I have to do something special to get Cyberduck working or is this configuration not supported?

host i-* mi-*
ProxyCommand sh -c "aws ssm start-session --target %h --document-name AWS-StartSSHSession --parameters 'portNumber=%p'"

**Comment by dkocher** [comment]
Only the `ProxyJump` configuration directive is supported [^1].

[^1]: https://docs.cyberduck.io/protocols/sftp/#connect-via-ssh-tunnel-through-bastion-server
